examples/dmd/nonlinear_elasticity.py

```diff
@@ -0,0 +1,629 @@
+'''
+   libROM MFEM Example: nonlinear_elasticity (adapted from ex10p.cpp)
+
+
+      This examples solves a time dependent nonlinear elasticity
+      problem of the form dv/dt = H(x) + S v, dx/dt = v, where H is a
+      hyperelastic model and S is a viscosity operator of Laplacian
+      type.
+
+   How to run:
+      mpirun -np 8 python <arguments>
+
+      Example 1 of arguments:
+      nonlinear_elasticity.py -s 2 -rs 1 -dt 0.01 -tf 5 -visit
+   
+      Output 1:
+      Relative error of DMD velocity (v) at t_final: 5.000000 is 1.145E-03
+      Relative error of DMD position (x) at t_final: 5.000000 is 7.371E-05
```

Comment:
I am getting following in quartz:

Relative error of DMD velocity (v) at t_final: 5.000000 is 2.501E-03
Relative error of DMD position (x) at t_final: 5.000000 is 1.325E-04

Have you run this in quartz as well?